pkg/plugin/signalhandler/handler.go

```diff
@@ -24,8 +24,9 @@ import (
 	"context"
 	"os"
 	"os/signal"
-	"sync/atomic"
 	"syscall"
+
+	"go.uber.org/atomic"
```

Comment:
Not sure this chance should be addressed by lint, wdyt?

Reply:
I think `sync/atomic` is enough, but we can use `go.uber.org/atomic` because it has more features than `sync/atomic`.

If we use `sync/atomic`, it's better not to mix `sync/atomic` and `go.uber.org/atomic` because it's confusing.

Currently, we have the rule below.
https://github.com/pipe-cd/pipecd/blob/40f8375b0266a087d19bb3cf0de4945550fa429d/.golangci.yml#L44-L45

Reply:
Okay, I got your point 👍